EIPS/eip-2612.md

```diff
@@ -4,7 +4,8 @@ title: permit – 712-signed approvals
 description: ERC-20 approvals via secp256k1 signatures
 author: Martin Lundfall (@Mrchico)
 discussions-to: https://github.com/ethereum/EIPs/issues/2613
-status: Review
+status: Last Call
+last-call-deadline: 2022-07-30
 type: Standards Track
 category: ERC
 created: 2020-04-13
```

Comment:
Just to confirm, is it the author's intention to *not* follow EIP-165? I believe it's the author's judgement call but just to make sure it's a conscious decision.